MAUI/LinearGauge/scale.md

```diff
@@ -0,0 +1,362 @@
+---
+layout: post
+title: Axis in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing Axis of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+
+documentation: ug
+---
+
+# Axis in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The Linear Gauge axis is a scale where a set of values can be plotted. An axis can be customized by changing the thickness, color and edge styles. Axis elements such as labels and ticks can also be easily customized and you can also inverse the axis.
+
+## Default axis
+
+By default axis will have the [`Minimum`]() axis value as 0 and the [`Maximum`]() axis value as 100. Without any changes the default axis of the Linear Gauge will be displayed as follows. 
```

Comment:
By default, axis will have the [`Minimum`]() axis value as 0 and the [`Maximum`]() axis value as 100. Without any changes, the default axis of the Linear Gauge will be displayed as follows.